recipe/session/recipeImplementation.go

```diff
@@ -37,8 +39,64 @@ var protectedProps = []string{
 	"antiCsrfToken",
 }
 
+var JWKCacheMaxAgeInMs = 60000
+var JWKRefreshRateLimit = 500
+var jwksFunctions []sessmodels.GetJWKSResult
```

Comment:
also add a test to make sure that this variable is updated when the jwks endpoint is recalled